src/docs/commands_keybindings.md

```diff
@@ -1,122 +1,138 @@
 ---
-title: Commands and Keybindings
+title: Commands/Menus/Keybindings
 ---
 
-# Commands and Keybindings
+# Commands, Menus and Keybindings
 
-Theia can be extended in a number of different ways. Commands allow packages to contribute unique commands that can then be called by other packages. It's also possible to add keybindings and contexts to these commands, so that they can only be called under certain conditions (window focus, current selection etc.).
+Commands are runnable actions defined by an ID, a name and the function to be executed. Commands can be triggered via the quick access bar, they can be bound to keybindings or menu items and they can be called programmatically. Commands can be context sensitive, so that they can only be called under certain conditions (window focus, current selection etc.).
```

Comment:
> Commands are runnable actions defined by an ID, a name and the function to be executed.

This is not entirely true given a command's definition:

```ts
export interface Command {
    /**
     * A unique identifier of this command.
     */
    id: string;
    /**
     * A label of this command.
     */
    label?: string;
    originalLabel?: string;
    /**
     * An icon class of this command.
     */
    iconClass?: string;
    /**
     * A category of this command.
     */
    category?: string;
    originalCategory?: string;
}
```

If we are to mention the `name` (label) we should mention it is **optional**.

Reply:
correct, i adapted the text